File: FrontEnd/src/app/models/respuesta.ts
export class Respuesta{
    id?:number;
    descripcion?:string;
    esCorrecta?:boolean;
    constructor(description:string,esCorrecta:boolean,id?:number) {
        this.descripcion=description;
        this.esCorrecta=esCorrecta;
        this.id=id;
    }
}

File: Back End/Demo.Entities/Respuesta.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Demo.Entities
{
    public class Respuesta:EntityBase
    {
        [Required]
        [Column(TypeName = "varchar(150)")]
        public string Descripcion { get; set; }
        [Required]
        public bool EsCorrecta { get; set; }
        public int PreguntaId { get; set; }
        public Pregunta? Pregunta { get; set; }
    }
}
